### PR TITLE
Manage changelogs

### PR DESCRIPTION
**Pull request by Dani21**
Supports admin-side PR by adding a "notify" column to changelogs, allowing editing and manual creation of changelogs, and validations to support maintenance.

### CODE REVIEW

**Review by corywright — APPROVED** [review]
Approving but will wait for @paulkva's review before merging.

File: service-war/src/main/java/com/redshiftsoft/tesla/web/mvc/changelog/ChangeLogController.java
Comment (paulkva) [inline service-war/src/main/java/com/redshiftsoft/tesla/web/mvc/changelog/ChangeLogController.java:106]: I thought we were moving in the opposite direction? Why did this delete endpoint change from POST to GET?

Comment (Dani21) [reply service-war/src/main/java/com/redshiftsoft/tesla/web/mvc/changelog/ChangeLogController.java:106]: Dang it, good catch. I think I changed it to POST and back separately in this commit while testing changes, but I thought the rebase would catch this.

**Review by paulkva — APPROVED** [review]

**Pull request merged**
